test_common/harness/imageHelpers.cpp

```diff
@@ -990,7 +990,8 @@ float get_max_relative_error(const cl_image_format *format,
 #else
     {
 #if !defined(_WIN32)
-#warning Implementations will likely wish to pick a max allowable sampling error policy here that is better than the spec
+//#warning Implementations will likely wish to pick a max allowable sampling
+// error policy here that is better than the spec
```

Comment:
This is worth some discussion. My assumption had been that this warning is desired to alert implementers about the lack of coverage. It would be better to allow it to be set externally so no code modifications are required, but that's out of the scope here.